js/src/forum/ForumApplication.js

```diff
@@ -68,9 +68,11 @@ export default class ForumApplication extends Application {
    * An object which stores previously searched queries and provides convenient
    * tools for retrieving and managing search values.
    *
-   * @type {SearchState}
+   * @type {HeaderSearchState}
    */
-  search = new SearchState();
+  search = new HeaderSearchState();
+
+  searchResults = false;
```

Comment:
This isn't used anywhere. I assume this was before you added the static `hasSearchResults` property to page ?

Reply:
I am an idiot and was looking at an individual commit. Ignore me.